### ISSUE TITLE
b2_contact.cpp lines 123 & 124 (b2Assert) are nearly right

### ISSUE THREAD

**Issue by HughPH** [erincatto/box2d#603]
The lines read
```c
	b2Assert(0 <= typeA && typeB < b2Shape::e_typeCount);
	b2Assert(0 <= typeA && typeB < b2Shape::e_typeCount);
```

I expect they should probably be 
```c
	b2Assert(0 <= typeA && typeA < b2Shape::e_typeCount);
	b2Assert(0 <= typeB && typeB < b2Shape::e_typeCount);
```

**Comment by Hexlord** [comment]
I expect the same, should we prepare a PR? @erincatto

**Comment by erincatto** [comment]
Fixed in #616
